### ISSUE TITLE
The loss turns to Nan while traing on faster rcnn/detr/deformable detr and so on

### ISSUE THREAD

**Issue by 1184125805** [open-mmlab/mmdetection#8024]
![QQ图片20220522132407](https://user-images.githubusercontent.com/38450078/169680166-57279177-aa08-44d9-a26c-8b42722356a1.png)
I can train my dataset in yolov3_d53_mstrain-608_273e_coco.py.But can not reach any mAP when I use faster-rcnn,detr and deformable detr.It's the same dataset.I tried to set lr to very low,but it can not work,even the clip.

**Comment by ZwwWayne** [comment]
It seems that the loss_cls is very high and the accur is low during trainning. There might be some issues in the config or data, which requires more information to diagnose.
Could you follow the issue template to provide more.information?

**Comment by 1184125805** [comment]
I change the "load from" from "absolute Path" to "Path from Content Root",and change samples_per_gpu from 1 to 2,also I set the auto_scale_lr to True. It seems everything is okay.
But I have another question now,Why the acc is so high from the beginning?It's normal?
![QQ图片20220524094400](https://user-images.githubusercontent.com/38450078/169931702-1eb694a0-219c-46e1-981e-b090a1555d94.png)

**Comment by ZwwWayne** [comment]
> I change the "load from" from "absolute Path" to "Path from Content Root",and change samples_per_gpu from 1 to 2,also I set the auto_scale_lr to True. It seems everything is okay.
> 
> But I have another question now,Why the acc is so high from the beginning?It's normal?
> 
> ![QQ图片20220524094400](https://user-images.githubusercontent.com/38450078/169931702-1eb694a0-219c-46e1-981e-b090a1555d94.png)
> 
> 

yes it is normal since you use pretrained model.
